### PR TITLE
updated docker-compose to include mongodb instructions - Issue #68

### PR DESCRIPTION
**Pull request by misterdabu**
Related issues: [please specify]

## Description:
Updated the docker-compose file to include instructions for mongodb from https://hub.docker.com/_/mongo

## I solemnly swear that:
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code in hard-to-understand areas
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works **<- N/A**
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: database/localhost/docker-compose.yaml [hunk 1]
```diff
@@ -17,3 +17,17 @@ services:
...
+    image: mongo:latest
+    container_name: tilde-mongodb
+    restart: always
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: root
+      MONGO_INITDB_ROOT_PASSWORD: password
+      MONGO_INITDB_DATABASE: root-db
+    volumes:
+      - ./gitignore/mongodb:/var/lib/mongodb/data
+      - ./docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d
```
Comment (sheenarbw) [inline database/localhost/docker-compose.yaml:31]: I'm not convinced that these volumes are as they should be.
Firstly:
```
      - ./gitignore/mongodb:/var/lib/mongodb/data
```
Are you sure that mongo stores it's data in `/var/lib/mongodb/data` ? I dont think it does.  Take a look at the mondo image documerntation to see what volumnes it suggests: https://hub.docker.com/_/mongo  

Also, what is this line for?
```
      - ./docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d
```
If you look inside the directory it just has sql stufff. https://github.com/Umuzi-org/Tilde/tree/develop/database/localhost/docker-entrypoint-initdb.d 

And I expect mongo will completely ignore it anyway, unless it's mimicking the postgres image for some reason. 

It's really not good to make guesses with this kind of thing.

**Review by sheenarbw — CHANGES_REQUESTED** [review]
I think you made some guesses when configuring the volumes. 

In the face of ambiguity, refuse the temptation to guess -  That's a line from the "Zen of Python". It's very good advice.

**Comment by misterdabu** [comment]
So sorry about that, I had copied and pasted with a replace and didn't realize it. That was my fault. I have updated the volume tested (see below) and resubmitted it. Thank you.

image

**Comment by asradebe** [comment]
Hi there. Seems this PR has been open for quite some.

I will be closing off this PR and read as unsuccessful and you will be unassigned to the issue you were working on. Whenever you feel ready to collaborate with us again please head on to https://discord.com/channels/758708091583332382/762669571852468254 to get further assistance. We thank you so much for your participation and collaboration. 

Our Sincerest Thanks
The Team

**Comment by misterdabu** [comment]
I am so sorry. I didn't realize that my second commit didn't get merged. Was there something wrong with the second commit? I idid test to make sure both of the databases were running simultaneously. Thanks you for letting me know.

**Pull request closed without merging**